packages/mds-metrics-sheet/metrics-log.ts

```diff
@@ -78,6 +79,26 @@ async function appendSheet(sheetName: string, rows: MetricsSheetRow[]) {
   }
 }
 
+function eventCountsToStatusCounts(events: { [s in VEHICLE_EVENT]: number }) {
+  return Object.keys(events).reduce(
```

Comment:
Starting with `(Object.keys(events) as VEHICLE_EVENT[]).reduce` would allow you to get rid of all the other `as VEHICLE_EVENT` casts.